### PR TITLE
Italian Translation

### PR DESCRIPTION
**Pull request by Thebigbot0000**
I translated namae.dev in italian, cool project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,6 +45,7 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
     <td align="center"><a href="https://github.com/miguelsmuller"><img src="https://avatars.githubusercontent.com/u/4589909?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Miguel Müller</b></sub></a><br /><a href="#translation-miguelsmuller" title="Translation">🌍</a></td>
     <td align="center"><a href="https://atzu.ml"><img src="https://avatars.githubusercontent.com/u/32654584?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Azurnex</b></sub></a><br /><a href="#translation-z3ro0k" title="Translation">🌍</a></td>
     <td align="center"><a href="http://snazzah.com"><img src="https://avatars.githubusercontent.com/u/7025343?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Snazzah</b></sub></a><br /><a href="https://github.com/uetchy/namae/commits?author=Snazzah" title="Code">💻</a></td>
+    <td align="center"><a href="http://thebigbot.mod.land"><img src="https://avatars.githubusercontent.com/u/77632836?s=48&amp;v=4" width="100px;" alt=""/><br /><sub><b>Thebigbot</b></sub></a><br /><a href="https://github.com/uetchy/namae/commits?author=Thebigbot0000" title="Code">🌍</a></td>
```
Comment (uetchy) [inline README.md:48]: You don't have to do this yourself. We'll add you here anyways after your PR gets merged.

File: public/locales/it/translation.json [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  "about": "Più informazioni su di noi",
+  "available": "Disponibile",
+  "blog": "Articoli del blog",
+  "community": "Comunità",
+  "contributors": "Contributori",
+  "countryCode": "it",
+  "description": "Verifica la disponibilità dei nomi che vuoi dare a i tuoi progetti  in tutti i registri principali contemporaneamente.",
+  "gettingStarted": "Inizia",
+  "gettingStartedWithExample": "Prova con namae",
+  "issues": "Utilizzi",
```
Comment (uetchy) [inline public/locales/it/translation.json:11]: Is this the correct translation for (GitHub) Issues?

File: README.md (hunk 1 above)
Comment (Thebigbot0000) [reply README.md:48]: By tonight I send it to you without that piece of code (European time)

File: public/locales/it/translation.json (hunk 2 above)
Comment (Thebigbot0000) [reply public/locales/it/translation.json:11]: Yes, yes

File: README.md [hunk 3]
```diff
@@ -45,7 +45,7 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
     <td align="center"><a href="https://github.com/miguelsmuller"><img src="https://avatars.githubusercontent.com/u/4589909?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Miguel Müller</b></sub></a><br /><a href="#translation-miguelsmuller" title="Translation">🌍</a></td>
     <td align="center"><a href="https://atzu.ml"><img src="https://avatars.githubusercontent.com/u/32654584?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Azurnex</b></sub></a><br /><a href="#translation-z3ro0k" title="Translation">🌍</a></td>
     <td align="center"><a href="http://snazzah.com"><img src="https://avatars.githubusercontent.com/u/7025343?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Snazzah</b></sub></a><br /><a href="https://github.com/uetchy/namae/commits?author=Snazzah" title="Code">💻</a></td>
-  </tr>
+
```
Comment (uetchy) [inline README.md:48]: You accidentally removed the ending tr tag

File: public/locales/it/translation.json (hunk 2 above)
Comment (uetchy) [reply public/locales/it/translation.json:11]: Hmm. Quick dictionary lookup leads me to 'Uses'. Can you provide some Italian blog posts that include the translated word for 'GitHub Issues'? Just to clarify.

**Review by uetchy — CHANGES_REQUESTED** [review]

File: public/locales/it/translation.json (hunk 2 above)
Comment (Thebigbot0000) [reply public/locales/it/translation.json:11]: Ah, I was wrong now I correct with Problemi

**Comment by Thebigbot0000** [comment]
I cancel as soon as possible the commit for credits in the readme

**Pull request closed without merging**